app/full_text_search.py

```diff
@@ -0,0 +1,87 @@
+import logging
+import re
+
+from google.appengine.api import search as appengine_search
+
+import model
+
+# The index name for full text search
+PERSON_FULLTEXT_INDEX_NAME = 'person_information'
```

Comment:
Nit, but it would be nice to be consistent whether to spell it "fulltext" or "full_text". Currently this constant is FULLTEXT but the file name is full_text.